paas_app_charmer/app.py

```diff
@@ -192,6 +206,18 @@ def _app_layer(self) -> ops.pebble.LayerDict:
         services[self._workload_config.service_name]["override"] = "replace"
         services[self._workload_config.service_name]["environment"] = self.gen_environment()
 
+        for service_name, service in services.items():
+            # Add environment variables to all worker processes.
+            if service_name.endswith(WORKER_SUFFIX):
```

Comment:
do you think we should be case-insensitive here?

Reply:
I have updated it so the worker name suffix can be case insensitive